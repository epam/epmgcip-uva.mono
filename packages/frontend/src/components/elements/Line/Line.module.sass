@use 'src/style/colors'

.line
   width: 100%
   height: 1px
   background-color: colors.$gray-200
