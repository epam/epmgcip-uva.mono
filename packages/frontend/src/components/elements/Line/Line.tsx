import css from './Line.module.sass';

export const Line = () => {
  return <span className={css.line}></span>;
};
