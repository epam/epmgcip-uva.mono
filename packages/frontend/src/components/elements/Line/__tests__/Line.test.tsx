import { describe, expect, it } from 'vitest';
import { render } from '@testing-library/react';
import { Line } from '../Line';

describe('Testing: Line', () => {
  it('should render', () => {
    const { container } = render(<Line />);

    expect(container).toMatchSnapshot();
  });
});
