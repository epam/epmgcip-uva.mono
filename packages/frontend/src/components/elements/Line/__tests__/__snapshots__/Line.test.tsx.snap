// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Testing: Line > should render 1`] = `
<div>
  <span
    class="_line_d53aaf"
  />
</div>
`;
